cg/services/order_validation_service/workflows/mutant/models/sample.py

```diff
@@ -26,17 +27,45 @@ class MutantSample(Sample):
     control: Annotated[ControlEnum, BeforeValidator(parse_control)] = ControlEnum.not_control
     elution_buffer: Annotated[ElutionBuffer, BeforeValidator(parse_buffer)]
     extraction_method: Annotated[ExtractionMethod, BeforeValidator(parse_extraction_method)]
+    _lab_code: str = PrivateAttr(default="SE100 Karolinska")
     organism: str
     original_lab: OriginalLab
+    original_lab_address: str
     pre_processing_method: PreProcessingMethod
     primer: Primer
     priority: PriorityEnum
     quantity: int | None = None
     reference_genome: str
     region: Region
+    region_code: str
     selection_criteria: SelectionCriteria
     _verified_organism: str | None = PrivateAttr(default=None)
 
+    @model_validator(mode="before")
+    @classmethod
+    def set_original_lab_address(cls, data: any):
+        if isinstance(data, dict):
+            is_set = bool(data["original_lab_address"])
+            data["original_lab_address"] = (
+                data["original_lab_address"]
+                if is_set
+                else ORIGINAL_LAB_ADDRESSES[data["original_lab"]]
+            )
```

Comment:
```suggestion
            if bool(data["original_lab_address"]):
                data["original_lab_address"] = data["original_lab_address"]
            else:
                data["original_lab_address"] = ORIGINAL_LAB_ADDRESSES[data["original_lab"]]
```

Reply:
does this make sense though? `data["original_lab_address"] = data["original_lab_address"]`?
Maybe
```suggestion
            if not bool(data["original_lab_address"]):
                data["original_lab_address"] = ORIGINAL_LAB_ADDRESSES[data["original_lab"]]
```

Reply:
Clarified it